packages/material-ui/package.json

```diff
@@ -44,7 +44,7 @@
     "@material-ui/utils": "^4.1.0",
     "@types/react-transition-group": "^2.0.16",
     "clsx": "^1.0.2",
-    "convert-css-length": "^2.0.0",
+    "convert-css-length": "^2.0.1",
```

Comment:
Do we need to force the update?

Reply:
https://github.com/KyleAMathews/convert-css-length/pull/2 is only included in ^2.0.1

Reply:
I guess we don't have to fear any duplicate in people bundle, so forcing the update is fine.